dali/sasha/saxref.cpp

```diff
@@ -2266,14 +2266,20 @@ class CSashaXRefServer: public ISashaServer, public Thread
         // NB: must be a list of planes only
         ForEachItemIn(i1, list) {
             const char *planeName = list.item(i1);
-            Owned<IPropertyTree> plane = getStoragePlane(planeName);
-            if (isContainerized()) {
-                if (!plane)
-                    throw makeStringExceptionV(-1, LOGPFX "Unknown data plane name: %s", planeName);
-                groups.append(planeName);
-                cnames.append(planeName);
+            Owned<IPropertyTreeIterator> planesIter = getPlanesIterator("data", planeName);
+            ForEach(*planesIter) {
+                IPropertyTree &plane = planesIter->query();
+                bool isNotCopy = !plane.getPropBool("@copy", false);
+                bool isNotHthorPlane = !plane.getPropBool("@hthorplane", false);
+                if (isNotCopy && isNotHthorPlane) {
```

Comment:
minor style (possibly personal).  I would prefer isCopy = ... isHthorPlane and the not to be in the test:

  if (!isCopy && !isHThorPlane)

idiomatically it is easier to read.